runut.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+usage() {
+        USAGE="Usage:
+-a : To run all the s3ut test cases.
+-f : To provide gtest filter.
+     Run with –f \"TestSuit.Testcase\" to run specific test suit or test case.
+     Ex 1. Run specific test suit -
+        runut.sh –f \"S3GetObjectActionTest.*\"
+
+     Ex 2. Run specific test case -
+        runut.sh –f \"S3GetObjectActionTest.ValidateObjectOfSizeZero\"
+-h : For usage."
+        echo -e "$USAGE"
+	exit 0
+}
+
+
+filter=
+extra_opts=
+while getopts :fh opt; do
+	case "$opt" in
+	a)
+		;;
+	f)
+		shift
+		filter="$1"
+		if [ -z "$filter" ]; then
+			echo -e "\nMust specify some UT pattern with -f option\n"
+			usage
+		fi
+		extra_opts="--gtest_filter=$filter"
+		;;
+	h)
+		usage
+		;;
+	*)
+		usage
+		;;
+	esac
+done
+
+export LD_LIBRARY_PATH="$(pwd)/third_party/motr/motr/.libs/:"\
+"$(pwd)/third_party/motr/helpers/.libs/:"\
+"$(pwd)/third_party/motr/extra-libs/gf-complete/src/.libs/:"\
+"$(pwd)/third_party/motr/extra-libs/galois/src/.libs/:"
+
+echo "Running UTs..."
+./bazel-bin/s3ut --gtest_repeat=1 "$extra_opts"
```

Comment:
Why is `--gtest_repeat=1` necessary? That seems to be the default, to only run the tests once.

Reply:
removed --gtest_repeat=1